source/release-notes/backend.txt

```diff
@@ -13,15 +13,35 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240306:
+
+6 March 2024 Release
+~~~~~~~~~~~~~~~~~~~~
+
+- Created a new endpoint to link multiple data sources to an existing App.
+
+- Updated the timeout variable, ``maxTimeMS``, to accept ``float64`` for ``db.aggregate``
+  and ``db.coll.aggregate`` commands. 
+
+- Added a check to ensure custom user data is not enabled before deleting a 
+  datasource. If custom user data is enabled, the user will get a 400 error.
+
+- If sync is paused or terminated, schema version information will still be shown in App Services UI.
```

Comment:
Since we're referring to the product name here, I think it should be capitalized.
```suggestion
- If Sync is paused or terminated, schema version information will still be shown in App Services UI.
```